### DISCUSSION TITLE
Any ideas?

### DISCUSSION THREAD

**Discussion by saltyshiomix** [saltyshiomix/nextron#139] [Ideas]
Feel free to tell us about your ideas!

**Comment by Psycokwet** [comment]
I don't know if that's a thing to say here, but I'll try : 
Currently, in my project, I have some dll/dylib/so files, from a package I'm managing and using, and when calling it from build, It doesn't work with the files from the asar archives, I'm forced to go seek them from asar unpacked, if you see what I mean? Maybe we could do something about it? I'm not sur what, but, I'm just saying ^^

Also, I have some mac development, and I'm still very lost with the licensing and info.plist, maybe we could make it easier to set the licensing and set the authorization from plist? 

Thank you for your work anyway ! n_n

**Comment by Psycokwet** [comment]
Also ! Is it okay if I extend sample code for IPC communication ? As I was starting I was blindly keeping up the same way you did on the sample, but got caught up by issues when using multiple time the same key (it's obvious now, but as I was only starting it was less obvious) in multiple components.
